charts/incubator/watcharr/templates/_configmap.tpl

```diff
@@ -0,0 +1,22 @@
+{{/* Define the configmap */}}
+{{- define "watcharr.configmaps" -}}
+{{- $secretName := (printf "%s-watcharr-secrets" (include "tc.v1.common.lib.chart.names.fullname" $)) }}
+
+{{- $watcharr := .Values.watcharr -}}
+
+{{- $secretKey := randAlphaNum 64 -}}
+
+ {{- with lookup "v1" "Secret" .Release.Namespace $secretName -}}
+   {{- $secretKey = index .data "JWT_SECRET" | b64dec -}}
+ {{- end }}
+watcharr-env:
+  enabled: true
+  data:
+    .env: |
```

Comment:
there is no need to put it in the .env file.

Any library that reads the env, is basically configuring the actual system env.
Also with this, the lookup will not work.